### PR TITLE
gh-93433: Fix dis doc example output

### LINKED ISSUE #93433: `dis` doc output is inaccurate for 3.11
**Documentation**

The first example and subsequent examples containing LOAD_GLOBAL in the dis docs need updating to reflect what 3.11 actually outputs.

### PR DESCRIPTION
**Pull request by Fidget-Spinner**
Fixes #93433

### CODE REVIEW

File: Doc/library/dis.rst [hunk 1]
```diff
@@ -42,11 +42,11 @@ the following command can be used to display the disassembly of
    >>> dis.dis(myfunc)
      1           0 RESUME                   0
 
-     2           2 PUSH_NULL
```
Comment (kumaraditya303) [inline Doc/library/dis.rst:45]: Would it be possible to use doctest here ?

Comment (Fidget-Spinner) [reply Doc/library/dis.rst:45]: I think we'd need to `import dis` to do that.

Comment (sweeneyde) [reply Doc/library/dis.rst:45]: I think we can do this at the top of the module:

```
.. testsetup::

   import dis
```

That's what [the itertools docs do](https://github.com/python/cpython/blob/3.10/Doc/library/itertools.rst).

Comment (Fidget-Spinner) [reply Doc/library/dis.rst:45]: Fascinating, I had no clue that existed. TIL, thanks!

**Review by kumaraditya303 — APPROVED** [review]
LGTM, It seems that 3.11 would require manual backport.

**Pull request merged**